arch/arm/boot/dts/overlays/hd44780-lcd-overlay.dts

```diff
@@ -0,0 +1,46 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+    compatible = "brcm, bcm2708";
```

Comment:
Although in practice this string isn't checked, it should at least be one of the recognised strings. These days I would just use: "brcm,bcm2835" (note the lack of a space after the comma).